.eslintrc.js

```diff
@@ -21,6 +21,7 @@ module.exports = {
   ],
   "rules": {
     "no-var": "error",
+    "no-case-declarations" : "warn",
```

Comment:
why we need to modify the lint rule? here is a tip for how to refactor https://eslint.org/docs/latest/rules/no-case-declarations to refactor

Reply:
got